users/views.py

```diff
@@ -1,3 +1,30 @@
-from django.shortcuts import render
+import re, json, bcrypt, jwt
 
-# Create your views here.
+from django.http     import JsonResponse
+from django.views    import View
+
+from .models         import User
+from FRA_WE_BACK.settings import SECRET_KEY
+
+class LoginView(View):
+    def post(self, request):
+        try:
+            data           = json.loads(request.body)
+            user_email     = data['email']
+            user_password  = data['password']
+            user           = User.objects.get(email=user_email)
+
+            if bcrypt.checkpw(user_password.encode('utf-8'), user.password.encode('utf-8')):
+                return JsonResponse({"message": "invalid_password"}, status=401)
+
+            token = jwt.encode ({'id':user.id}, SECRET_KEY, algorithm='HS256')
```

Comment:
알고리즘도 환경변수로 관리 해주시면 좋을 것 같습니다
```suggestion
            from FRA_WE_BACK.settings import SECRET_KEY, ALGORITHM
            '''
            token = jwt.encode ({'id':user.id}, SECRET_KEY, algorithm=ALGORITHM)
```

Reply:
수정해주시고 conflict 해결 후 push 해주세요~

Reply:
충돌 해결 수정했습니다.